src/metrics.rs

```diff
@@ -104,13 +104,24 @@ struct Stats {
     fds: usize,
 }
 
+#[cfg(feature="sysconf")]
+fn ticks_per_second() -> Result<f64> {
+    sysconf::raw::sysconf(sysconf::raw::SysconfVariable::ScClkTck)
+        .map_err(|_| "failed to get _SC_CLK_TCK".into())
+        .map(|x| x as f64)
+}
+
+#[cfg(not(feature="sysconf"))]
+fn ticks_per_second() -> Result<f64> {
+    Err("could not get _SC_CLK_TCK - not built with `--features sysconf`".into())
```

Comment:
Since this causes `parse_stats` to unconditionally fail, maybe gate that one as well? Not sure how prometheus works, wouldn't this flood the log with error messages?